lib/cli/resolve-input.js

```diff
@@ -16,7 +16,7 @@ const baseArgsSchema = {
 };
 
 const resolveArgsSchema = (commandOptionsSchema) => {
-  const options = { boolean: new Set(), string: new Set(), alias: new Map() };
+  const options = { boolean: new Set(), string: new Set(), alias: new Map(), multiple: new Set() };
```

Comment:
Required for https://github.com/serverless/serverless/pull/9131/files#diff-04014f23e713e4f682d271931cd3c22d9a08022610fde64d5154ced65b7f5cc5L26